src/nvim/lua/executor.c

```diff
@@ -1840,11 +1840,11 @@ void nlua_set_sctx(sctx_T *current)
   xfree(info);
 }
 
-void nlua_do_ucmd(ucmd_T *cmd, exarg_T *eap)
+int nlua_do_ucmd(ucmd_T *cmd, exarg_T *eap, bool preview)
```

Comment:
Cross-referencing hints make the codebase much more understandable.

```suggestion
/// @param preview Invoke the callback as a |:command-preview| handler.
int nlua_do_ucmd(ucmd_T *cmd, exarg_T *eap, bool preview)
```